beacon-chain/blockchain/service_test.go

```diff
@@ -94,11 +94,12 @@ func setupBeaconChain(t *testing.T, beaconDB db.Database) *Service {
 		DepositContainers: []*ethpb.DepositContainer{},
 	})
 	require.NoError(t, err)
-	web3Service, err = powchain.NewService(ctx, &powchain.Web3ServiceConfig{
-		BeaconDB:        beaconDB,
-		HttpEndpoints:   []string{endpoint},
-		DepositContract: common.Address{},
-	})
+	web3Service, err = powchain.New(
```

Comment:
Not sure about switching to just "New". It kind of implies the service is the only thing in the package, and we often have other types in a service package to support the service. I think `NewService` is fine and could be consistent across services.